Assets/HoloToolkit/Input/Tests/Shaders/button_unlit_shader.shader

```diff
@@ -52,7 +52,7 @@
 			struct v2f
 			{
 				float4 pos : SV_POSITION;
-				fixed4 uv : TEXCOORD0;
+				fixed3 uv : TEXCOORD0;
```

Comment:
There was an additional warning here because the output wasn't being fully written in the vertex shader. We only write to uv.xy and uv.z, but uv.w wasn't used.